$NetBSD: patch-device_crypto_crypt-base64.cpp,v 1.1 2022/08/05 14:50:29 he Exp $

Make dependence on signed chars explicit, to allow build
on NetBSD/powerpc where chars are unsigned unless qualified.

--- device/crypto/crypto-base64.cpp.orig	2008-09-05 20:01:47.000000000 +0000
+++ device/crypto/crypto-base64.cpp
@@ -36,7 +36,7 @@
 
 // See RFC3548 for details of Base64 Encode / Decode standards
 
-static const char base64string[] = {62, -1, -1, -1, 63, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, -1, -1, -1, 0, -1, -1, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, -1, -1, -1, -1, -1, -1, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51};
+static const signed char base64string[] = {62, -1, -1, -1, 63, 52, 53, 54, 55, 56, 57, 58, 59, 60, 61, -1, -1, -1, 0, -1, -1, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, -1, -1, -1, -1, -1, -1, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48, 49, 50, 51};
 
 
 // Base64 character check
